guides/common/modules/proc_configuring-direct-ad-integration.adoc

```diff
@@ -0,0 +1,56 @@
+[id="configuring-the-active-directory-authentication-source-on-projectserver_{context}"]
+= Configuring the Active Directory authentication source on {ProjectServer}
+
+Enable Active Directory (AD) users to access {Project} by configuring the corresponding authentication provider on your {ProjectServer}.
+
+.Procedure
+. Configure the System Security Services Daemon (SSSD) to use the AD access control provider to evaluate and enforce Group Policy Object (GPO) access control rules for the `foreman` PAM service:
+.. In your `/etc/sssd/sssd.conf` file, add the following lines to the `domain` section for the AD domain:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+[domain/_EXAMPLE_]
+access_provider = ad
```

Comment:
This line seems to be already there after doing `realm join`

Reply:
Removed.